specs/import_cluster_workflow_ui.adoc

```diff
@@ -0,0 +1,147 @@
+
+== Proposed change
+
+Front-end related changes:
+
+	* When trigger on "import cluster", We need to be add one popup for selecting storage service(gluster/ceph).
+
+	* And breadcrumb need to be add in top of the page for tracking the location information.
+
+	* In case of unexpected or error result, we need to be add error message as suggested by UX design.
+	( https://redhat.invisionapp.com/share/R88EUSGJK#/screens/193145484 )
+
+	* After selecting appropriate storage service(gluster/ceph), Now we have two step to go for import cluster
+
+		** Step 1: Need to be populate all node list in tabular format as UX design suggested. so for tabular component we should have generic component(later can be used for others) for generate the tabular view. ( https://redhat.invisionapp.com/share/R88EUSGJK#/screens/183602753 )
+
+		** Step 2: We should have template for populate the cluster details (cluster name, storage service and node details ) which will come after selecting node from previous step.
+		( https://redhat.invisionapp.com/share/R88EUSGJK#/screens/193145495 )
+
+	* We need to populate error message in case of ceph if selected node is not a monitor.
+	( https://redhat.invisionapp.com/share/R88EUSGJK#/screens/197663177 )
+
+	
+
+=== Alternatives
+
+The solution discussed in the 'Proposed change' is a two step process(involves two API calls). Instead this can be combined into a single API as well where the API taken a single node as input and rest executed in one go. The downside with approach would be that the user will not a chance to verify the fetched cluster details are correct. Also there is an impact on the proposed UX design as well
+
+=== Data model impact:
+
+See 'Impacted Modules' section where all the details are populated.
+
+=== Impacted Modules:
+
+==== Tendrl frontend impact:
+
+   * Need to be add one more routing for import cluster.
+   * And error message need to be implement accross the application as a common component.
+   * This changes will impect unit test case as well. need to be update the test case according code changes.
+
+==== Tendrl Backend impact:
+
+This is discussed in a different specification - https://github.com/Tendrl/specifications/issues/54
+
+
+=== Security impact:
+
+None.
+
+=== Other end user impact:
+
+There should be changes in UI as well and so end user as well impacted.
+
+=== Performance impact:
+
+
+None.
+
+=== Other deployer impact:
+
+
+None.
+
+=== Developer impact:
+
+
+None.
+
+
+== Implementation:
+
+Created the github issues to track the implementation:
+
+* https://github.com/Tendrl/tendrl_frontend/issues/63
+
+=== Assignee(s):
+
+
+Primary assignee:
+  kamleshverma1
+
+Other contributors:
+  gnehapk
+
+=== Work Items:
+
+Basic import cluster flows are already implemented in UI. The changes with respect to auto discovering the cluster details as part import cluster needs to be implemented from UI.
+
+
+== Dependencies:
+
+* https://github.com/Tendrl/specifications/issues/54
+
```

Comment:
Any other dependencies? UX design reviewed and signed off? Any dependency on patternfly or any UX components?